packages/components/src/badges/Badge.tsx

```diff
@@ -0,0 +1,25 @@
+import { cn } from "@spin-spot/utils";
+interface BadgeProps {
+  labelName?: string;
+  label?: string;
+  leftIcon?: React.ReactNode;
+  className?: string;
+}
+
+export function Badges({ labelName, label, leftIcon, className }: BadgeProps) {
```

Comment:
Cambiar el nombre del componente a Badge
```suggestion
export function Badge({ labelName, label, leftIcon, className }: BadgeProps) {
```